pkg-py/src/querychat/__init__.py

```diff
@@ -1,14 +1,17 @@
-from querychat._greeting import greeting
-from querychat.querychat import (
-    init,
-    sidebar,
-    system_prompt,
-)
-from querychat.querychat import (
-    mod_server as server,
-)
-from querychat.querychat import (
-    mod_ui as ui,
-)
+from . import express
+from ._deprecated import greeting, init, sidebar, system_prompt
+from ._deprecated import mod_server as server
+from ._deprecated import mod_ui as ui
+from ._querychat import QueryChat
 
-__all__ = ["greeting", "init", "server", "sidebar", "system_prompt", "ui"]
+__all__ = (
+    "QueryChat",
+    "express",
```

Comment:
Should we include `express` in `__all__`? I kind of think it'd be better to require explicitly reaching into `querychat.express`.


```suggestion
    "QueryChat",
```

Reply:
By including `express` here, it makes this possible:

```python
import querychat

querychat.express.QueryChat
```

or, somewhat equivalently:

```python
from querychat import *

express.QueryChat
```

It's the same reason why we include `reactive`, `ui`, etc in `shiny/__init__.py`...

Reply:
Yeah I mean I don't like either of those and would prefer to discourage them. I think it's much better to have to explicitly import from `express` because you generally don't want both things in the same session. It's different from `reactive` and `ui` in that way.

This seems like the right way to import the express variant:

```python
from querychat.express import Querychat
```

or equivalently

```python
from querychat.express import *
```

and this seems also good to me

```python
import querychat.express

querychat.express.QueryChat
```